From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 15 Mar 2019 22:52:10 +0100
Subject: [PATCH] {MAYBE UPS} drivers/usb/typec: add missing const qualifier
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

gcc complains:

    drivers/usb/typec/tps6598x.c: In function ‘tps6598x_write_4cc’:
    drivers/usb/typec/tps6598x.c:176:40: error: passing argument 3 of
    ‘tps6598x_block_write’ discards ‘const’ qualifier from pointer
    target type [-Werror=discarded-qualifiers]
      return tps6598x_block_write(tps, reg, val, sizeof(u32));
                                            ^~~
---
 drivers/usb/typec/tps6598x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/tps6598x.c b/drivers/usb/typec/tps6598x.c
index c674abe3cf99..31fe62d3c4a2 100644
--- a/drivers/usb/typec/tps6598x.c
+++ b/drivers/usb/typec/tps6598x.c
@@ -127,7 +127,7 @@ tps6598x_block_read(struct tps6598x *tps, u8 reg, void *val, size_t len)
 }
 
 static int tps6598x_block_write(struct tps6598x *tps, u8 reg,
-				void *val, size_t len)
+				const void *val, size_t len)
 {
 	u8 data[TPS_MAX_LEN + 1];
 
-- 
